lib/resources/tasks/run.js

```diff
@@ -1,22 +1,10 @@
 import gulp from 'gulp';
 import browserSync from 'browser-sync';
 import historyApiFallback from 'connect-history-api-fallback/lib';
+import { CLIOptions } from 'aurelia-cli';
```

Comment:
We don't use spaces elsewhere in the framework ([example](https://github.com/aurelia/framework/blob/master/src/aurelia.js#L5)) so we should follow that standard